pkg/kudoctl/cmd/init_integration_test.go

```diff
@@ -226,7 +231,8 @@ func TestIntegInitWithServiceAccount(t *testing.T) {
 	// Test Case 1, the serviceAccount does not exist, expect serviceAccount not exists error
 	err = cmd.run()
 	require.Error(t, err)
-	assert.Equal(t, "error installing: Service Account test-account does not exists - KUDO expects the serviceAccount to be present in the namespace sa-integration-test\n", err.Error())
+	assert.Equal(t, "failed to verify installation requirements", err.Error())
```

Comment:
Since you're at it: this `TestIntegInitWithServiceAccount` is way too big for a test (>100 slocs) and it gets even bigger now. Do you mind either using a table with test cases or just individual test cases?

Reply:
I've reworked that a bit. Not too easy, as the tests are kind of based on each other, but it's a bit better now.